### DISCUSSION TITLE
How can I handle DB Migrations through Quart?

### DISCUSSION THREAD

**Discussion by Jean-Lytehouse** [pallets/quart#199] [Q&A] [answered]
Hey there,

I'm in the process of migrating from Flask to Quart. I use Flask with SQLAlchemy on my postgres DB, and use Flask CLI to handle my DB migrations (flask db migrate, flask db upgrade, etc etc). 

Is it possible to do this through Quart? If so, how is that accomplished? If not, is there an extension that I can use to accomplish this, or another possible solution? I've been banging my head against this all day haha

Best,
Jean-vicente

**Comment by pgjones** [comment, marked as answer]
I don't think there is an exact equivalent extension for Quart. However, I recommend [Quart-DB](https://github.com/pgjones/quart-db) which has migrations as [documented](https://quart-db.readthedocs.io/en/latest/how_to_guides/migrations.html).
